server/Quiz.ts

```diff
@@ -0,0 +1,53 @@
+import EventEmitter from "https://deno.land/x/events/mod.ts";
+import { v4 } from "https://deno.land/std/uuid/mod.ts";
+import Student from "./Student.ts";
+
+class Quiz extends EventEmitter {
+    id_from_lecturer: String;
+    server_id: String;
+    time_seconds: number;
+    questions: any;
+    student_ids: String[];
+    student_answers: Map<String, any>;
+    active: boolean;

```

Comment:
Any Reason why you using here snake_case instead of camelCase?
![obraz](https://user-images.githubusercontent.com/26521377/116002573-adba3200-a5fa-11eb-8c0b-d3dcb063c7f0.png)

Reply:
i wanted to distinguish fields from payloads from variables in code, maybe it`s wrong